tests/Unit/MonkeyStubs.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Automattic\LegacyRedirector\Tests\Unit;
+
+use Brain\Monkey;
+
+final class MonkeyStubs {
+
+	public function __construct() {
+		Monkey\Functions\stubs(
+			array(
+				'wp_parse_url' => static function ( $url, $component ) {
+					return parse_url( $url, $component );
+				},
+			)
+		);
+	}
```

Comment:
Rather than a class which is just used to add new stubs, I wonder if it makes more sense to create a custom unit test case (`WPStubbedTestCase`), like https://github.com/Yoast/wp-test-utils/blob/develop/src/BrainMonkey/YoastTestCase.php and perhaps extended from it to save having to re-define the WP stubs it already defines.

That way, instead of `final class UtilsTest extends use Yoast\WPTestUtils\BrainMonkey\TestCase {` it would be `final class UtilsTest extends use Automattic\LegacyRedirector\Tests\Unit\WPStubbedTestCase {`

That would then allow removal of the `UtilsTest::set_up()` as well.

Reply:
makes sense, I am now extending class `YoastTestCase` from `MonkeyStubs` and I could remove the `UtilsTest::set_up()`